src/components/workflowEditor/workflowEditor.tsx

```diff
@@ -416,9 +520,6 @@ class WorkflowEdit extends Component<WorkflowEditProps, WorkflowEditState> {
                                     getWorkflows={this.getWorkflows}
                                     refreshParentWorkflows={this.props.getWorkflows}
                                     envIds={this.state.envIds}
-                                    isLastNode={
```

Comment:
Yes since it was used for validating if there are any other children for the delete request and we have removed the condition so this check was not used anymore